### ISSUE TITLE
Typings about WebRTC

### ISSUE THREAD

**Issue by leaysgur** [facebook/flow#4366]
I can't find type definitions about WebRTC like `RTCPeerConnection`.

Is there a plan to support or some progress?

**Comment by nkt** [comment]
Any updates?
Maybe you can open-source your [private definitions](https://github.com/facebook/fbjs/blob/master/packages/eslint-config-fbjs/shared.js#L83)?

**Comment by ea167** [comment]
Is the lack of such obvious types as WebRTC ones for several years 
the sign that flow project is being abandoned?

**Comment by gkz** [comment]
> Is the lack of such obvious types as WebRTC ones for several years
> the sign that flow project is being abandoned?

No, it means no one has put up a PR for them (and no one has requested these internally). We regularly merge PRs which add new library definitions.

**Comment by jcready** [comment]
Does the fact that there are more than 100 bugs with the [needs triage](https://github.com/facebook/flow/issues?q=is%3Aissue+sort%3Aupdated-desc+is%3Aopen+label%3A%22needs+triage%22) label mean the project is abandoned? Because it certainly feels that way.

**Comment by SamChou19815** [comment]
The libdef has been moved to flow-typed. If you want to make improvements, please make PR over there.
